lms/validation/authentication/_launch_params.py

```diff
@@ -5,11 +5,12 @@
 from lms.models import LTIUser, display_name
 from lms.services import ApplicationInstanceNotFound, LTILaunchVerificationError
 from lms.validation._base import PyramidRequestSchema
+from lms.validation._lti import LTIAuthParamsSchema
 
 __all__ = ("LaunchParamsAuthSchema",)
 
 
-class LaunchParamsAuthSchema(PyramidRequestSchema):
+class LaunchParamsAuthSchema(LTIAuthParamsSchema, PyramidRequestSchema):
```

Comment:
I think this could be moved to `lms/validation/authentication/_lti.py` and renamed to `LTI11AuthSchema` along side the LTI13 contuerpart.

Left is as it for now.